packages/catalog-model/examples/all-components.yaml

```diff
@@ -18,3 +18,4 @@ spec:
     - ./components/wayback-archive-ingestion-component.yaml
     - ./components/wayback-archive-storage-component.yaml
     - ./components/wayback-search-component.yaml
+    
```

Comment:
whoops, let's get rid of this! That will help with the changeset check as well
```suggestion
```

Reply:
Sure thing, working on a rebase to remove that changeset error along with those linter errors that i received.